### ISSUE TITLE
Extracting training/validation accuracy

### ISSUE THREAD

**Issue by LSBigum** [fizyr/keras-retinanet#1019]
How would I go about extracting the training and validation accuracy? With accuracy I'm referring to the classifications - whether an image containing a label has been predicted to contain that label by the network with a certainty above a cutoff-value. I only have one class in my data set, but I feed the network a lot of negative examples as well (meaning photos not containing any labels), so I would like to see how often the network recognizes that the object is/isn't in the image.

**Comment by hgaiser** [comment]
If you want the training accuracy (we use the mAP measurement), you will have to make some custom modifications, we don't compute that by default. For validation accuracy, check the [--val-annotations](https://github.com/fizyr/keras-retinanet/blob/master/keras_retinanet/bin/train.py#L387) argument (assuming you're using a CSV dataset).
